lib/Ohmbrewer_Screen.h

```diff
@@ -213,6 +213,11 @@ namespace Ohmbrewer {
              */
             Menu* _homeMenu;
 
+           /**
```

Comment:
Changed the mainMenu to be a local variable.

Created an issue for the destructor bit